carla/models/catalog/catalog.py

```diff
@@ -56,25 +60,27 @@ def __init__(
         else:
             raise Exception("Model type not in catalog")
 
-        self._model = load_model(model_type, data.name, ext, cache, models_home, **kws)
+        self._model: Any = load_model(
```

Comment:
Not sure if its adding a lot of value adding `Any` annotations to properties? What do we gain from that?

Reply:
Nothin 😄 I tested a bit with typehintin and forget to remove that.